tools/emprofile.py

```diff
@@ -15,75 +15,64 @@
 
 profiler_logs_path = os.path.join(tempfile.gettempdir(), 'emscripten_toolchain_profiler_logs')
 
-OUTFILE = 'emprofile.' + time.strftime('%Y%m%d_%H%M')
-for i in range(len(sys.argv)):
-  arg = sys.argv[i]
-  if arg.startswith('--outfile=') or arg.startswith('-o='):
-    OUTFILE = arg.split('=', 1)[1].strip().replace('.html', '')
-    sys.argv[i] = ''
-  elif arg == '-o':
-    OUTFILE = sys.argv[i + 1].strip().replace('.html', '')
-    sys.argv[i] = sys.argv[i + 1] = ''
-
 
 # Deletes all previously captured log files to make room for a new clean run.
 def delete_profiler_logs():
-  try:
+  if os.path.exists(profiler_logs_path):
     shutil.rmtree(profiler_logs_path)
-  except IOError:
-    pass
 
 
 def list_files_in_directory(d):
   files = []
-  try:
-    items = os.listdir(d)
-    for i in items:
+  if os.path.exists(d):
+    for i in os.listdir(d):
       f = os.path.join(d, i)
       if os.path.isfile(f):
-        files += [f]
-    return files
-  except IOError:
-    return []
+        files.append(f)
```

Comment:
Somewhat indifferent about changing `files += [f]` to `files.append(f)`, though is there a specific reason to favor append instead of the += form? iiuc the `array += [element]` syntax has been extensively used in the past in this repo?

Reply:
Yeah I don't feel strongly about it.   

I think its a good habit to get into when building lists in python because it avoids creating the extra lists each time and (sometimes importantly) avoids re-binding the variable each time.     (When you do `+=` you end up throwing away both the old list and the single element list on the RHS.  Not that is performance not relevant in this particular case :)